contrib/kafka/config/500-webhook.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafka-webhook
+  namespace: knative-eventing
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: kafka-webhook
+      role: kafka-webhook
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels: *labels
+    spec:
+      serviceAccountName: kafka-webhook
+      containers:
+      - name: kafka-webhook
+        terminationMessagePolicy: FallbackToLogsOnError
+        image: github.com/knative/eventing/contrib/kafka/cmd/webhook
+        volumeMounts:
+        - name: config-logging
+          mountPath: /etc/config-logging
+        env:
+        - name: SYSTEM_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CONFIG_LOGGING_NAME
+          value: config-logging
+        - name: WEBHOOK_NAME
+          value: kafka-webhook
+        resources:
+          limits:
+            # TODO set a proper value.
+            memory: 1000Mi   # An initial guess, but consistent with serving
```

Comment:
not sure what you mean; see the webhook here:

https://github.com/knative/serving/blob/master/config/webhook.yaml#L45-L51

I am not sure we should add blindly any of these values ...

Reply:
it was a copy/paste from eventing webhook. I think we need to update that one as well. Look here:
https://github.com/knative/eventing/blob/4484d2d1461202f4eba133f8ab79f6f37619f1f6/config/500-webhook.yaml#L53

I agree, let's remove the limits for now and then we see.

Reply:
Done